components/dashboard/src/settings/selectClass.tsx

```diff
@@ -48,41 +48,24 @@ export default function SelectWorkspaceClass(props: SelectWorkspaceClassProps) {
                     Choose the workspace machine type for your workspaces.
                 </p>
                 <div className="mt-4 space-x-3 flex">
-                    <SelectableCardSolid
-                        className="w-36 h-32"
-                        title="Standard"
-                        selected={workspaceClass === "standard"}
-                        onClick={() => actuallySetWorkspaceClass("standard")}
-                    >
-                        <div className="flex-grow flex items-end p-1">
-                            <svg width="112" height="64" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                <path
-                                    d="M0 8a8 8 0 0 1 8-8h16a8 8 0 1 1 0 16H8a8 8 0 0 1-8-8ZM0 32a8 8 0 0 1 8-8h16a8 8 0 1 1 0 16H8a8 8 0 0 1-8-8ZM0 56a8 8 0 0 1 8-8h16a8 8 0 1 1 0 16H8a8 8 0 0 1-8-8ZM40 6a6 6 0 0 1 6-6h60a6 6 0 0 1 6 6v28a6 6 0 0 1-6 6H46a6 6 0 0 1-6-6V6Z"
-                                    fill="#D6D3D1"
-                                />
-                            </svg>
-                        </div>
-                    </SelectableCardSolid>
-                    <SelectableCardSolid
-                        className="w-36 h-32"
-                        title="XL"
-                        selected={workspaceClass === "XL"}
-                        onClick={() => actuallySetWorkspaceClass("XL")}
-                    >
-                        <div className="flex-grow flex items-end p-1">
-                            <svg width="112" height="64" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                <path
-                                    d="M0 8a8 8 0 0 1 8-8h16a8 8 0 1 1 0 16H8a8 8 0 0 1-8-8ZM40 6a6 6 0 0 1 6-6h60a6 6 0 0 1 6 6v28a6 6 0 0 1-6 6H46a6 6 0 0 1-6-6V6Z"
-                                    fill="#D9D9D9"
-                                />
-                                <path
-                                    d="M84 0h22a6 6 0 0 1 6 6v28a6 6 0 0 1-6 6H68L84 0ZM0 32a8 8 0 0 1 8-8h16a8 8 0 1 1 0 16H8a8 8 0 0 1-8-8Z"
-                                    fill="#78716C"
-                                />
-                                <path d="M0 56a8 8 0 0 1 8-8h16a8 8 0 1 1 0 16H8a8 8 0 0 1-8-8Z" fill="#D9D9D9" />
-                            </svg>
-                        </div>
-                    </SelectableCardSolid>
+                    <WorkspaceClass
+                        additionalStyles="w-68 h-32"
+                        selected={workspaceClass === "g1-standard"}
+                        onClick={() => actuallySetWorkspaceClass("g1-standard")}
+                        category="GENERAL PURPOSE"
+                        friendlyName="Standard"
+                        description="Up to 4 vCPU, 8GB memory, 30GB disk"
+                        powerUps={1}
+                    />
+                    <WorkspaceClass
+                        additionalStyles="w-68 h-32"
```

Comment:
**suggestion**: Minor width fix for better spacing.

```suggestion
                        additionalStyles="w-80 h-32"
```

| BEFORE | AFTER |
|-|-|
| <img width="785" alt="option-before" src="https://user-images.githubusercontent.com/120486/179555564-32832b3d-ddea-404f-aeae-3b580724a68c.png"> | <img width="785" alt="option-after" src="https://user-images.githubusercontent.com/120486/179555567-4bc21be1-9a7e-4ec5-8db8-bb8fdf02072b.png"> |